Simplify copy on Install page (#669)
jupyter/jupyter.github.io: install.md

---
layout: page
title: Installing Jupyter
tagline: Get up and running on your computer
permalink: /install
---

> Project Jupyter's tools are available for installation via the [Python Package Index](https://pypi.org/), the leading repository of software created for the Python programming language. Its offerings can be installed with a variety of package management tools that run via a [command-line interface](https://en.wikipedia.org/wiki/Command-line_interface), including [conda](https://docs.conda.io/), [mamba](https://mamba.readthedocs.io/) and [pipenv](https://pipenv.pypa.io/).

Reviewer: I suggest that we note the existence of many tools, but keep our code instructions to the simplest possible tool, which I believe is pip (especially now that JupyterLab has federated extensions, so there's no javascript stuff needed when you install from pip). Technically I believe pypa recommends pip as the tool for installation anyway: https://packaging.python.org/en/latest/guides/tool-recommendations/#installation-tool-recommendations

> Which one you use is up to you. Pipenv is recommended by the [Python Packaging Authority](https://www.pypa.io/). Conda and mamba are preferred by many members of [PyData](https://pydata.org/) and can by acquired via [miniforge](https://github.com/conda-forge/miniforge#mambaforge).

Reviewer: ```suggestion
This page uses instructions with [pip](https://pip.pypa.io/en/stable/) as it is [the recommended installation tool for Python](https://packaging.python.org/en/latest/guides/tool-recommendations/#installation-tool-recommendations).
If you require _environment management_ as opposed to just installation, you should look into [conda](https://docs.conda.io/), [mamba](https://mamba.readthedocs.io/), and [pipenv](https://pipenv.pypa.io/).

> **note**: If you install JupyterLab with `conda` or `mamba`, we recommend using [the `conda-forge` channel](https://conda-forge.org/).
```

Author: Should the  note go in the Jupyter Lab section, maybe below the install line, rather that in the top section?

## JupyterLab

> Install JupyterLab with one of the following:

Reviewer: ```suggestion
Install JupyterLab with `pip`:
```

```bash

> pipenv install jupyterlab

Reviewer: ```suggestion
pip install jupyterlab
```

Reviewer: Could we leave the installation steps to just `pip`? I think having multiple installation options will confuse people, assuming that most people that install jupyter by following the instructions on this page are novices who are less-experienced with Python in general. The less information we need to give them to get started, the better.

mamba install -c conda-forge jupyterlab
conda install -c conda-forge jupyterlab
```

Once installed, launch JupyterLab with:

```bash
jupyter-lab
```

## Jupyter Notebook

Install the classic Jupyter Notebook with one of the following:

```bash

> pipenv install notebook
> conda install -c conda-forge notebook
> mamba install -c conda-forge notebook

Reviewer: ```suggestion
pip install notebook
```

```

To run the notebook:

```bash
jupyter notebook
```

## Voilà

Install Voilà with one of the following:

```bash

> pipenv install voila

Reviewer: ```suggestion
pip install voila
```

> mamba install -c conda-forge voila

Reviewer: Ugh I can't suggest edits for lines that were removed - if you agree w/ the change we can manually remove the extra lines
